.github/workflows/build.yml

```diff
@@ -53,5 +53,131 @@ jobs:
           key: maven-repo-${{ runner.os }}-${{ steps.get-date.outputs.date }}
 
       - name: Build with Maven
-        run: mvn -B formatter:validate verify -Dnative --file pom.xml
+        run: mvn -B formatter:validate install -Dnative --file pom.xml
 
+      - name: Kubernetes KinD Cluster
+        uses: container-tools/kind-action@v1
+        with:
+          version: v0.11.1
+          registry: true
+      - name: Install OPM and Operator SDK tool
+        uses: redhat-actions/openshift-tools-installer@v1
+        with:
+          source: "github"
+          opm: "1.20.0"
+          operator-sdk: "1.17.0"
+
+      - name: Install Operator Lifecycle Manager and Operator SDK into Kind
+        run: operator-sdk olm install
+
+      - name: Run Joke sample using Quarkus DEV mode
+        run: |
+          SCRIPTS=$(pwd)/.github/scripts
+          K8S_NAMESPACE=dev
+
+          # Create and set namespace
+          kubectl create namespace $K8S_NAMESPACE
+          kubectl config set-context --current --namespace=$K8S_NAMESPACE
+
+          # Build manifests
+          cd samples/joke
+          mvn install
+
+          # Install CRDs
+          kubectl apply -f target/manifests/jokerequests.samples.javaoperatorsdk.io-v1.yml
```

Comment:
Alternatively, the operator could be configured to automatically apply the generated CRD if missing or changed using the `quarkus.operator-sdk.crd.apply=true` property.

Reply:
These steps mimic what is described in the quickstart section in samples/joke/README.md. 
Do you want me to also update the README.md by using the property `quarkus.operator-sdk.crd.apply=true`?

Reply:
I tried to use this property but it didn't work. I've opened an issue with this: https://github.com/quarkiverse/quarkus-operator-sdk/issues/242. Let me know if you agree with the problem and if you want me to fix it.

Reply:
Indeed, I wasn't clear in my request/suggestion 😄 
We could either keep things as they are or add the property to `application.properties` (or as a parameter when launching the dev mode). As explained in #242, the property only applies to dev mode at the moment.

Reply:
I tried to use it when launching the dev mode but still didn't work as I'm running the command in the background. So when the resource is created, the dev mode command has not installed the CRD yet. 
I could overcomplicate the script to wait until the dev mode command installs the CRD, but I'd prefer to install the CRD separately to keep this script simpler if you don't mind.

Reply:
Marking this suggestion as resolved. We can do this change once https://github.com/quarkiverse/quarkus-operator-sdk/issues/242 is implemented in a later pull request.